website/docs/cloud-docs/users-teams-organizations/api-tokens.mdx

```diff
@@ -49,6 +49,16 @@ Organization API tokens have permissions across the entire organization. They ca
 
 -> **Note:** This endpoint cannot be accessed with [organization tokens](#organization-api-tokens). You must access it with a [user token](/terraform/cloud-docs/users-teams-organizations/users#api-tokens) or [team token](#team-api-tokens).
 
+## Audit Trails Tokens
+
+Tokens may be generated for reading organization audit trails. They are intended to be used in the [HCP Terraform for Splunk](/terraform/cloud-docs/integrations/splunk) app.
+
+To manage the token for reading audit trails for a specific organization, go to **Organization settings > API Token** and use the controls under the "Audit Token" header.
```

Comment:
```suggestion
To manage an organization's audit trails token, go to **Organization settings > API Token** and use the settings under the "Audit Token" header.
```

Reply:
Is the header for "Audit Token" plural or singular in the UI?

Reply:
It's singular!